t/fnv_test.c

```diff
@@ -0,0 +1,27 @@
+#include "test/test.h"
+#include "utils/fnv.h"
+
+int main()
+{
+    Fnv32_t (*fnv)(void *,size_t) = fnv_init("fnv32a_str");
+    Fnv32_t (*fold16)(Fnv32_t) = fold_init("fold16");
+    Fnv32_t (*fold1)(Fnv32_t) = fold_init("fold1");
+    Fnv32_t (*fold)(Fnv32_t) = fold_init("fold_noop");
+    Fnv32_t res = 0;
+    pretty_assert(fnv !=  NULL);
+    pretty_assert(fold16 !=  NULL);
+    pretty_assert(fold1 !=  NULL);
+
+    // hurz hashes to 0x60bdfa92
```

Comment:
Where is this magic value coming from?

Reply:
My code, but it checks out:
https://md5calc.com/hash/fnv1a32/hurz